### DISCUSSION TITLE
Is it possible to show all logs on the same panel?

### DISCUSSION THREAD

**Discussion by yacinehmito** [F1bonacc1/process-compose#215] [Q&A]
Sometimes, I'd like to show all processes at once. Is it possible with process-compose?

**Comment by F1bonacc1** [comment]
Hi @yacinehmito,
It is not natively possible in PC, but there is an option to write to a [unified log file](https://f1bonacc1.github.io/process-compose/logging/#merge-into-a-single-file-unified-logging), which you can `tail -f` in another process in PC.
